css/at-rules/font-face.json

```diff
@@ -0,0 +1,283 @@
+{
+  "css": {
+    "properties": {
+      "font-face": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/at-rules/font-face",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
```

Comment:
"12" (if it was supported in IE, and is in Edge, we can convert the true to "12", the first edge version)